docs/INSTALLATION.md

```diff
@@ -0,0 +1,233 @@
+# Installing OVE
+
+OVE needs to be installed before it can be used to control a display. OVE can be installed either by downloading and compiling the source code of the corresponding components or by running a specific installer available on the [**OVE Install**](https://github.com/ove/ove-install/) repository.
+
+All contributors to OVE are encouraged to download and compile the source code. All users of OVE are encouraged to use the OVE installers.
+
+## Installation by running OVE installers
+
+[**OVE Install**](https://github.com/ove/ove-install/) scripts are designed to install OVE into a [Docker](https://www.docker.com/get-started) environment.
+
+### Prerequisites
+
+* [git](https://git-scm.com/downloads)
+* [Docker](https://www.docker.com/get-started)
+* [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Configuring the environment
+
+The deployment environment needs to be pre-configured before running the OVE installers. This involves setting up ports and environment variables.
+
+The ports are pre-configured to a list of common defaults, but can be changed based on end-user requirements. Each port or port-range is defined as a mapping `HOST_PORT:CONTAINER_PORT`. Only the host ports can be changed, and it is important to note that **container ports must not be changed**.
+
+#### Environment variables
+
+Listed below are the environment variables that can be configured before installing OVE:
+
+* `OVE_HOST` - Hostname (or IP address + port) of OVE core
+* `TUORIS_HOST` - Hostname (or IP address + port) of the [Tuoris](https://github.com/fvictor/tuoris) service (dependency of SVG application).
+
+Listed below are the environment variables that can be configured before installing [**OVE Asset Services**](https://github.com/ove/ove-asset-services):
+
+* `MYSQL_RANDOM_ROOT_PASSWORD` - This variable is mandatory and must always be set to `yes`. This will generate a random initial password for the root user on the [MariaDB](https://mariadb.org/) database.
```

Comment:
Is this true? Can't the user instead chose a specific password, and expose it as `MYSQL_ROOT_PASSWORD`?

We should perhaps say "we **recommend** it be set to 'yes'".

Reply:
You can either set a root password or make it random, in our installers we make it random, and if you do not set it to 'yes' it will fail to install. Given this is a disposable container holding system state rather than an externally administered DB, I am wondering whether you need to go into such a lot of details with regards to the MariaDB configuration. But I can link to the configuration options as a resolution.

Reply:
They can choose to use the root user with a well-chosen password, but it's bad practice. I agree with you that his is our recommendation rather than something we enforce.

Reply:
I have changed the text accordingly to read as  "we recommend it be set to 'yes'".